include/RAJA/exec-agency/forall_agency.hxx

```diff
@@ -136,6 +136,51 @@ namespace RAJA
                         });
   }
 
+template <typename SEG_EXEC_POLICY, typename LOOP_BODY>
+RAJA_INLINE void forall(
+    IndexSet::ExecPolicy<agency_taskgraph_segit, SEG_EXEC_POLICY>,
```

Comment:
We probably want to have this work for both `agency::par` and `agency::omp::par`; we can probably just template `agency_taskgraph_segit`